### PR TITLE
googlemanagedprometheusexporter: add config option for cumulative normalization

### PR DESCRIPTION
**Pull request by ridwanmsharif**
This change will allow the configurer to set whether cumulative normalization should be set. The main benefit from this is so we can send the first point immediately as we see them.  This will allow our serverless offering to unfork the exporter:
https://github.com/GoogleCloudPlatform/run-gmp-sidecar/tree/main/collector/exporter/googlemanagedprometheusexporter

### CODE REVIEW

**Review by jsuereth — APPROVED** [review]

**Comment by ridwanmsharif** [comment]
@dashpole PTAL, I also don't have merge permissions so will wait for you to merge as well

**Review by dashpole — APPROVED** [review]
thanks!

**Pull request merged**
